blakserv/geometry.h

```diff
@@ -56,8 +56,8 @@ typedef struct V2
 
 // true if point (c) lies inside boundingbox defined by min/max of (a) and (b)
 #define ISINBOX(a, b, c) \
-   (fmin((a)->X, (b)->X) - EPSILON <= (c)->X && (c)->X <= fmax((a)->X, (b)->X) + EPSILON && \
-    fmin((a)->Y, (b)->Y) - EPSILON <= (c)->Y && (c)->Y <= fmax((a)->Y, (b)->Y) + EPSILON)
+   (fmin((a)->X, (b)->X) - 0.1f <= (c)->X && (c)->X <= fmax((a)->X, (b)->X) + 0.1f && \
+    fmin((a)->Y, (b)->Y) - 0.1f <= (c)->Y && (c)->Y <= fmax((a)->Y, (b)->Y) + 0.1f)
```

Comment:
Have you tried just redefining EPSILON to 0.1f in general?
Cause what you're 0.1f is doing here is what EPSILON is supposed to do....
Or at least may be add another EPSILON define ('EPSILONBIG') etc.

Since we're operating in 1:1024 roo fineness, anything below 1.0f is anyways 'very' small....